x-pack/plugins/observability_solution/dataset_quality/public/hooks/use_summary_panel.tsx

```diff
@@ -85,6 +85,9 @@ const useSummaryPanel = () => {
 
     isDatasetsActivityLoading,
     datasetsActivity,
+
+    numberOfDatasets: filteredItems.length,
+    numberOfDocuments: filteredItems.reduce((acc, curr) => acc + curr.degradedDocs.docsCount, 0),
```

Comment:
Would it be better if we count all docs here, not just the degraded ones? (Maybe `.totalDocs`?)

Reply:
That's the total number of documents in the timerange ([ref](https://github.com/elastic/kibana/blob/a9ae3e53f12576b6a360fcb714e94c89eaf3751b/x-pack/plugins/observability_solution/dataset_quality/common/data_streams_stats/data_stream_stat.ts#L30))

Reply:
I see, should this be outside of the `degradedDocs` property?

Reply:
I think this was done in such fashion because when querying the degradedDocs percentage we need the total number of documents in the timeRange. An improvement could be definitely extract this value from `degradedDocs` property when we are mapping the response to the state.

Reply:
Thanks for the background